### PR TITLE
Pr 2 fixes

### PR DESCRIPTION
**Pull request by prasanna1211**
Changed components as per `PR-2-fixes` in gatsby app.
1. Not very sure about Button. Please review the todo section. We need to discuss about the hover color

### CODE REVIEW

File: src/customStyleSystem.js [hunk 1]
```diff
@@ -18,11 +18,17 @@ const overflowY = style({
 const align = style({
   prop: 'align',
   cssProperty: 'text-align',
-})
+});
+
+const bk = style({
+  prop: 'bk',
+  cssProperty: 'background',
+});
```
Comment (Kameshwaran) [inline src/customStyleSystem.js]: I dont think we need this.
They have `background`

File: src/molecules/TextWithIcon/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ TextWithIcon.displayName = 'TextWithIcon';
 TextWithIcon.defaultProps = {
   position: 'absolute',
   backgroundSize: 'cover',
+  color: 'white',
```
Comment (Kameshwaran) [inline src/molecules/TextWithIcon/index.js]: @prasanna1211 why do we need `color` property for span?

Comment (Kameshwaran) [reply src/molecules/TextWithIcon/index.js]: You can do it like this, if you need color for the text.
<TextWithIcon>
  <Text.p color='white'>Test</Text.p>
</TextWithIcon>

File: src/atoms/Box/index.js [hunk 3]
```diff
@@ -49,6 +50,8 @@ const Box = styled.div`
   ${overflow}
   ${overflowX}
   ${overflowY}
+  ${bk}
```
Comment (Kameshwaran) [inline src/atoms/Box/index.js]: We can import `background` from styled-system.

**Comment by Kameshwaran** [comment]
@prasanna1211 Pull master branch and rebase your branch on it. 
I've already added `background` for 'Box'

**Comment by prasanna1211** [comment]
@Kameshwaran updated the changes.

File: src/atoms/Box/index.js [hunk 4]
```diff
@@ -53,6 +54,8 @@ const Box = styled.div`
   ${overflow}
   ${overflowX}
   ${overflowY}
+  ${background}
```
Comment (Kameshwaran) [inline src/atoms/Box/index.js]: duplicate `background` prop

Comment (prasanna1211) [reply src/atoms/Box/index.js]: updated. thanks for pointing.

**Pull request merged**